src/test/java/de/wejago/mqtt2influx/service/JsonSubscriberTest.java

```diff
@@ -0,0 +1,88 @@
+package de.wejago.mqtt2influx.service;
+
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.wejago.mqtt2influx.config.Device;
+import de.wejago.mqtt2influx.repository.InfluxDbRepository;
+import java.util.HashMap;
+import java.util.Map;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class JsonSubscriberTest {
+    @Mock
+    private ObjectMapper objectMapper;
```

Comment:
Don't mock it, but use a real ObjectMapper, that also saves some lines in the tests.